### PR TITLE
introduce short + easy-to-remember URLs for redirecting to external Community pages (fixes #340)

### LINKED ISSUE #340: Redirect `/slack` to A-Frame Slack sign-up page
https://aframe.io/slack should redirect to https://aframevr-slack.herokuapp.com

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: _config.prod.yml [hunk 1]
```diff
@@ -145,6 +145,20 @@ slack:
...
+  webvr:
+    url: https://www.reddit.com/r/webvr
+
+stack_overflow:
+  aframe:
+    url: https://stackoverflow.com/questions/tagged/aframe
+  webvr:
+    url: https://stackoverflow.com/questions/tagged/webvr
+
+awesome_list:
```
Comment (ngokevin) [inline _config.prod.yml]: seems like this could be under the github key

**Comment by ngokevin** [comment]
r+wc

File: _config.prod.yml (hunk 1 above)
Comment (cvan) [inline _config.prod.yml]: I like that better, thanks

**Comment by cvan** [comment]
thanks for the review

**Pull request merged**